cloudify_cli/commands/executions.py

```diff
@@ -526,3 +532,318 @@ def delete(logger, client, tenant_name, to_datetime, before, keep_last,
         logger.info('\nDeleted %d executions', deleted_executions_count)
     else:
         logger.info('\nNo executions to delete')
+
+
+@cfy.group(name='schedule')
+@cfy.options.common_options
+def schedule():
+    """Handle workflow execution scheduling [manager only]
+    """
+    pass
+
+
+@schedule.command(name='create',
+                  short_help='Schedule a workflow execution')
+@cfy.argument('name')
+@cfy.options.workflow_id(required=True)
+@cfy.options.deployment_id(required=True)
+@cfy.options.parameters
+@cfy.options.allow_custom_parameters
+@cfy.options.force(help=helptexts.FORCE_CONCURRENT_EXECUTION)
+@cfy.options.dry_run
+@cfy.options.wait_after_fail
+@cfy.options.queue
+@cfy.options.common_options
+@cfy.options.from_datetime(required=True, help="Time to schedule from.")
+@cfy.options.to_datetime(required=False, help="Time to schedule until.")
+@cfy.options.frequency
+@cfy.options.count
+@cfy.options.weekdays
+@cfy.options.rrule
+@cfy.options.slip
+@cfy.options.stop_on_fail
+@cfy.assert_manager_active()
+@cfy.options.tenant_name(required=False, resource_name_for_help='execution')
+@cfy.pass_client()
+@cfy.pass_logger
+def schedule_create(name,
+                    workflow_id,
+                    deployment_id,
+                    parameters,
+                    allow_custom_parameters,
+                    force,
+                    dry_run,
+                    wait_after_fail,
+                    queue,
+                    from_datetime,
+                    to_datetime,
+                    freq,
+                    count,
+                    weekdays,
+                    rrule,
+                    slip,
+                    stop_on_fail,
+                    tenant_name,
+                    client,
+                    logger):
+    """Schedule an execution of a workflow on a given deployment
+    """
+    utils.explicit_tenant_name_message(tenant_name, logger)
+    logger.info('Scheduling the execution of workflow `{0}` on deployment '
+                '`{1}`. Schuedule name: {2}'.format(workflow_id,
+                                                    deployment_id,
+                                                    name))
```

Comment:
```suggestion
    logger.info('Scheduling the execution of workflow `%s` on deployment '
                '`%s`. Schuedule name: %s',
                workflow_id, deployment_id, name))
```
the usual str.format + logging comment
(multiple occurences in this pr)